paddle/fluid/operators/sequence_ops/sequence_expand_op.cc

```diff
@@ -50,31 +51,47 @@ class SequenceExpandOp : public framework::OperatorWithKernel {
       auto& y_lod = y_var->Get<LoDTensor>().lod();
 
       PADDLE_ENFORCE_LE(x_lod.size(), 1UL,
-                        "Level number of Input(X)'s lod should not be "
-                        "greater than 1.");
-      PADDLE_ENFORCE_GT(y_lod.size(), 0UL,
-                        "Level number of Input(Y)'s lod should be "
-                        "greater than 0.");
-      PADDLE_ENFORCE(
+                        platform::errors::InvalidArgument(
+                            "Level of Input(X)'s lod should not be "
+                            "greater than 1. But received: lod level %u.",
+                            x_lod.size()));
+      PADDLE_ENFORCE_GT(
+          y_lod.size(), 0UL,
+          platform::errors::InvalidArgument(
+              "Level of Input(Y)'s lod should be greater than 0. But "
+              "received: lod level %u.",
+              y_lod.size()));
+      PADDLE_ENFORCE_EQ(
           ref_level == -1 ||
               (ref_level >= 0 && ref_level < static_cast<int>(y_lod.size())),
-          "Invlid `ref_level`, which should be either equal to -1 "
-          "or in [0, %d)",
-          y_lod.size());
+          true, platform::errors::InvalidArgument(
+                    "Invlid `ref_level`, which should be either equal to -1 "
+                    "or in [0, %d), but received `ref_level` = %u.",
+                    y_lod.size(), ref_level));
 
       if (ref_level == -1) ref_level = y_lod.size() - 1;
 
       if (x_lod.size() > 0) {
-        PADDLE_ENFORCE(x_lod[0].size() == y_lod[ref_level].size(),
-                       "Level number of Input(X)'s lod could be 0. Otherwise "
-                       "size of Input(X)'s first level lod should be equal to "
-                       "size of Input(Y)'s referred level lod.");
+        PADDLE_ENFORCE_EQ(
+            x_lod[0].size(), y_lod[ref_level].size(),
+            platform::errors::InvalidArgument(
+                "Level number of Input(X)'s lod could be 0. Otherwise "
+                "size of Input(X)'s first level lod should be equal to "
+                "size of Input(Y)'s referred level lod. But received: "
+                "Input(X).lod[0].size() = %u, Input(Y).lod[%d].size() = "
+                "%u",
+                x_lod[0].size(), y_lod[ref_level].size()));
```

Comment:
there are only 2 values, which is not enough for % holders?

Reply:
Done